kie-wb-parent/kie-wb-webapp/pom.xml

```diff
@@ -2144,7 +2152,7 @@
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>findbugs-maven-plugin</artifactId>
           <configuration>
-            <excludeFilterFile>${project.basedir}/src/main/findbugs/findbugs-exclude.xml</excludeFilterFile>
+            <excludeFilterFile>${project.basedir}/../../src/main/findbugs/findbugs-exclude.xml</excludeFilterFile>
```

Comment:
Would it not be better to put the "findbugs" configuration in this module?

I'm pretty sure this relative path crosses module boundaries?